From cc4c097a3e4d3b3ff7ffc9326f7285e0d2bcbd6c Mon Sep 17 00:00:00 2001
From: WingMan Kwok <w-kwok2@ti.com>
Date: Tue, 3 Oct 2017 19:47:19 -0400
Subject: [PATCH 101/102] ARM: dts: am57xx-idk-common: add prueth ptp time
 stamping tx interrupts

This patch adds the ptp tx interrupts required for the
support of IEEE-1588 PTP timestamping on the PRU-ICSS
ethernet ports.

Signed-off-by: WingMan Kwok <w-kwok2@ti.com>
---
 arch/arm/boot/dts/am57xx-idk-common.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/am57xx-idk-common.dtsi b/arch/arm/boot/dts/am57xx-idk-common.dtsi
index effd7ad..ad288bd 100644
--- a/arch/arm/boot/dts/am57xx-idk-common.dtsi
+++ b/arch/arm/boot/dts/am57xx-idk-common.dtsi
@@ -171,8 +171,8 @@
 		pruss2_emac0: ethernet-mii0 {
 			phy-handle = <&pruss2_eth0_phy>;
 			phy-mode = "mii";
-			interrupts = <20>, <22>;
-			interrupt-names = "rx", "tx";
+			interrupts = <20>, <22>, <23>;
+			interrupt-names = "rx", "tx", "ptp_tx";
 			/* Filled in by bootloader */
 			local-mac-address = [00 00 00 00 00 00];
 			/*tx-queue-size = /bits/ 16 <97 97 97 97 48>;*/
@@ -181,8 +181,8 @@
 		pruss2_emac1: ethernet-mii1 {
 			phy-handle = <&pruss2_eth1_phy>;
 			phy-mode = "mii";
-			interrupts = <21>, <23>;
-			interrupt-names = "rx", "tx";
+			interrupts = <21>, <23>, <24>;
+			interrupt-names = "rx", "tx", "ptp_tx";
 			/* Filled in by bootloader */
 			local-mac-address = [00 00 00 00 00 00];
 			/*tx-queue-size = /bits/ 16 <97 97 97 97 48>;*/
-- 
1.9.1

